content/influxdb/cloud-dedicated/reference/client-libraries/v3/python.md

```diff
@@ -181,22 +171,17 @@ Given that `write_client_options` isn't specified, the client uses the default [
 
 {{% code-placeholders "DATABASE_(NAME|TOKEN)" %}}
 
-<!-- Import for tests and hide from users.
-```python
+<!-- Import for tests and hide from users. -->
+<!--
 import os
```

Comment:
This allows pytest-codeblocks to run `import os` for test setup while hiding the irrelevant code from users.
When tests are run, pytest-codeblocks runs any `python` and shell codeblocks, including codeblocks within HTML comments.

Reply:
After you merge my PR into your branch and it updates here, then we can merge this PR.

Reply:
I’ve pushed the updates to my branch, and the PR should now reflects the latest changes